### PR TITLE
overflow visible post

### PR DESCRIPTION
**Pull request by rachelandrew**
A pre-announcement of a behavior change.

### CODE REVIEW

File: site/en/blog/overflow-replaced-elements/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+The `border-radius` property makes a square image look like a circle, but because `overflow` is visble the clipping no longer occurs.
+
+```css
+img {
+  border-radius: 50% 50%;
+  overflow: visible;
+}
+```
+
+Setting `inherit: all` and causing all elements to inherit, including `overflow`.
```
Comment (khushalsagar) [inline site/en/blog/overflow-replaced-elements/index.md:47]:
```suggestion
Setting `inherit: all` and causing all properties to inherit, including `overflow`.
```

File: site/en/blog/overflow-replaced-elements/index.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+```css
+img {
+  all: inherit;
+}
+```
+
+## Solution
+
+If this change means you see unwanted overflow, and overriding the overflow property to visible is not intentional, inspect the CSS applied to the element via DevTools. This should allow you to identify the CSS declaration which is overriding the overflow property to `visible` and update it to `clip`.
```
Comment (khushalsagar) [inline site/en/blog/overflow-replaced-elements/index.md:57]:
```suggestion
If this change means you see unwanted overflow, and overriding the overflow property to visible is not intentional, inspect the CSS applied to the element via DevTools. This should allow you to identify the CSS declaration which is overriding the overflow property to `visible` and remove it or update it to `clip`.
```

**Review by khushalsagar — COMMENTED** [review]
I don't have permission to hit "Approve" but this LGTM.

File: site/en/blog/overflow-replaced-elements/index.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+---
+title: A change to overflow on replaced elements in CSS
+description: >
+  An advanced warning of a change to CSS behavior that may cause unwanted overflow.
+subhead: >
+  An advanced warning of a change to CSS behavior that may cause unwanted overflow.
+layout: 'layouts/blog-post.njk'
+date: 2022-09-30
```
Comment (rachelandrew) [inline site/en/blog/overflow-replaced-elements/index.md:8]:
```suggestion
date: 2022-10-05
```

**Pull request merged**